bundle/regal/rules/imports/prefer_package_imports.rego

```diff
@@ -0,0 +1,63 @@
+# METADATA
+# description: Prefer importing packages over rules
+package regal.rules.imports["prefer-package-imports"]
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.regal.ast
+import data.regal.config
+import data.regal.result
+
+cfg := config.for_rule("imports", "prefer-package-imports")
+
+aggregate contains entry if {
+	imports_with_location := [imp |
+		some _import in input.imports
+
+		_import.path.value[0].value == "data"
+		count(_import.path.value) > 1
+		path := [part.value | some part in array.slice(_import.path.value, 1, count(_import.path.value))]
+
+		imp := object.union(result.location(_import), {"path": path})
+	]
+
+	entry := result.aggregate(rego.metadata.chain(), {
+		"imports": imports_with_location,
+		"package_path": [part.value | some i, part in input["package"].path; i > 0],
+	})
+}
+
+# METADATA
+# schemas:
+#   - input: schema.regal.aggregate
+aggregate_report contains violation if {
+	all_package_paths := {pkg |
+		some entry in input.aggregate
+		pkg := entry.aggregate_data.package_path
```

Comment:
Even if a file has no imports, its package path will still be set here right? I think this is the case, but just wanted to check that `all_package_paths` will be set to all packages, rather than all packages with imports.

Reply:
Yep!